tests/bitset.test.cpp

```diff
@@ -0,0 +1,202 @@
+#include <boost/ut.hpp>
+#include <cstdint>
+#include <libxbitset/bitset.hpp>
+
+namespace embed {
+boost::ut::suite bitset_test = [](){
+    using namespace boost::ut;
+
+    //TEST SET() FOR INITIAL VALUE OF 1
```

Comment:
```suggestion```

This comment is not necessary because the test name has this information